labcodes/django-react-webpack
# Update react helmet

**Link to task:** based on #57 , fix #56 

**How to test:**
- run the app
- verify no deprecation warnings are logged to the console

**Description of your solution:**
This PR upgrades react-helmet to v6. It's based on the react hooks PR, so be sure to review andmerge it before this one.
